letsencrypt/plugins/manual.py

```diff
@@ -82,10 +83,10 @@ def prepare(self):  # pylint: disable=missing-docstring,no-self-use
     def more_info(self):  # pylint: disable=missing-docstring,no-self-use
         return """\
 This plugin requires user's manual intervention in setting up a HTTP
-server for solving SimpleHTTP challenges and thus does not need to be
-run as a privilidged process. Alternatively shows instructions on how
-to use Python's built-in HTTP server and, in case of HTTPS, openssl
-binary for temporary key/certificate generation.""".replace("\n", "")
+ server for solving SimpleHTTP challenges and thus does not need to be
+ run as a privilidged process. Alternatively shows instructions on how
+ to use Python's built-in HTTP server and, in case of HTTPS, openssl
+ binary for temporary key/certificate generation.""".replace("\n", "")
```

Comment:
Since we are hiding this plugin from the UI, user's will likely never see this message, so it's probably better to move it to the docstring.

Could we also add information that this plugin can be run on laptop, while the code is executed on a remote web server?